tests/FilterTest.php

```diff
@@ -23,6 +23,37 @@ public function testParserIdentifiesSingleConditions()
         );
     }
 
+    public function testParserIdentifiesWildCardCharacters()
+    {
+        $expectedSimilar = QueryString::parse("foo=ba*");
+
+        $this->assertTrue(
```

Comment:
I'm sure there are special type assertions in php-unit ;)

Reply:
Used `assertInstanceOf()`, I guess that's what you wanted me to use.